src/components/breadcrumb-bar/utils/get-docs-breadcrumbs.ts

```diff
@@ -183,8 +185,7 @@ function findPathMatchedNodes(navNode, pathString, depth) {
 
 interface GetDocsBreadcrumbsOpts extends GetPathBreadcrumbsOpts {
 	baseName: string
-	productName: string
-	productPath: string
+	product: Pick<ProductData, 'slug' | 'name' | 'rootDocsPaths'>
```

Comment:
We now pass `ProductData`, rather than separate `productName` and `productPath`, so that we can get to `product.rootDocsPaths` as well (for our "nested" docs category breadcrumb logic).